NightscoutServiceKitTests/OTPManagerTestCase.swift

```diff
@@ -84,7 +84,7 @@ class OTPManagerTestCase: XCTestCase {
         //Arrange
         let testCoordinator = OTPTestCoordinator()
         let manager = testCoordinator.createOTPManager()
```

Comment:
Our OTPs should be 6 characters long. This test is for the wrong OTP, not invalid format.. I was providing the wrong input.